src/allmydata/scripts/run_common.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import print_function
```

Comment:
Note this is a renamed `tahoe_daemonize.py` with a few minor edits.  It doesn't get detected as a rename, I guess, because there is still a `tahoe_daemonize.py` (with much simpler contents).